contrib/galaxy.fedora-init

```diff
@@ -22,9 +24,11 @@ start() {
 	case "$(id -un)" in
 		$RUN_AS)
 			eval "$cmd"
+			touch /var/lock/subsys/galaxy
```

Comment:
It is quite possible that the `galaxy` user does not have permissions to write in `/var/lock/subsys/`.